kustomization/base/configMap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config
+  namespace: my-app-namespace
+data:
+  API_GATEWAY_URI: "api-gateway-service:7000"

```

Comment:
Any reason you had to hardcode the port number?

Reply:
Its listening to this port number.

Reply:
Please also include the protocol here.